### PR TITLE
CASMINST-6147 / CASMINST-6151

### PR DESCRIPTION
**Pull request by deantroe**
# Description

CASMINST-6147 Add note after Stage 0.4
CASMINST-6151 misc IUF doc updates

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/iuf/IUF.md [hunk 1]
```diff
@@ -591,7 +591,7 @@ INFO              BEGIN: preflight-checks(0)
 
 ### Log files
 
-IUF stores detailed information in log files which are stored on a Ceph Block Device typically mounted at `/etc/cray/upgrade/`. The default log file directory location can be overridden with the `iuf -b` and `iuf --log-dir`
+IUF stores detailed information in log files which are stored on a Ceph block device typically mounted at `/etc/cray/upgrade/`. The default log file directory location can be overridden with the `iuf -b` and `iuf --log-dir`
```
Comment (don-bahls-hpe) [inline operations/iuf/IUF.md:594]: `/etc/cray/upgrade/` is the directory where the mount point resides, but it looks to me like the actual root of the mount is `/etc/cray/upgrade/csm` (at least on tyr)

File: operations/iuf/workflows/configuration.md [hunk 2]
```diff
@@ -176,26 +178,29 @@ Once this step has completed:
...
-    - Configure system revision information via `sat setrev`
-  - SDU
-    - Configure SDU via `sdu setup`
-  - COS
-    - Set the COS root password in HashiCorp Vault
-  - UAN
-    - Set the UAN root password in HashiCorp Vault
+- COS
+  - Configure DVS and LNet with appropriate Slingshot settings
+  - Configure DVS and LNet for use on application nodes
```
Comment (don-bahls-hpe) [inline operations/iuf/workflows/configuration.md:187]: Maybe we should have say something like this.
* Configure DVS and LNet for use on application nodes (this is a change for this release)

File: operations/iuf/IUF.md [hunk 3]
```diff
@@ -591,7 +591,7 @@ INFO              BEGIN: preflight-checks(0)
...
 options (see `iuf -h` for details).
 
 Log files are organized by activity identifiers, for example `admin-230127`. The top-level `state` directory contains information internal to the implementation of IUF and is most often uninteresting to the administrator.
```
Comment (sfkramer) [inline operations/iuf/IUF.md:597]: ...and is inessential to the administrator.  (if this is true)

File: operations/iuf/workflows/management_rollout.md [hunk 4]
```diff
@@ -171,6 +171,20 @@ or `Management_Storage`. These values will be needed when upgrading NCN storage
         iuf -a "${ACTIVITY_NAME}" run -r management-nodes-rollout --limit-management-rollout Management_Master
         ```
 
+    1. Use `kubectl` to remove the `iuf-prevent-rollout=true` label from `ncn-m002`.
+
+        (`ncn-m001#`) Remove label from `ncn-m002`.
+
+        ```bash
+        kubectl label nodes "ncn-m002" --overwrite iuf-prevent-rollout-
```
Comment (sfkramer) [inline operations/iuf/workflows/management_rollout.md:179]: Is the dash needed after rollout in, --overwrite iuf-prevent-rollout**-**

File: operations/iuf/IUF.md (hunk 1 above)
Comment (deantroe) [reply operations/iuf/IUF.md:594]: Fixed in next commit.

File: operations/iuf/workflows/management_rollout.md (hunk 4 above)
Comment (deantroe) [reply operations/iuf/workflows/management_rollout.md:179]: Yes, that's the syntax used to clear a label.

**Review by don-bahls-hpe — APPROVED** [review]

File: operations/iuf/IUF.md [hunk 5]
```diff
@@ -638,7 +638,8 @@ products. `product_vars.yaml` is provided by HPE and the values are intended as
 Site variables, typically specified in a `site_vars.yaml` file, allow the administrator to override values provided by recipe
 variables, including global default entries and product-specific entries. HPE does not provide a `site_vars.yaml` file as it is
 strictly for site use cases. See the text at the top of the HPE-provided `product_vars.yaml` file for details on which override
-values can be specified in `site_vars.yaml`.
+values can be specified in `site_vars.yaml`. The `site_vars.yaml` file must reside on the Ceph block device typically mounted at
+`/etc/cray/upgrade/`.
```
[the remark this answers is not in the document]
Comment (deantroe) [reply operations/iuf/IUF.md]: Fixed in latest commit.

**Review by rustydb — APPROVED** [review]

**Comment by deantroe** [comment]
The release/1.4 version of this is https://github.com/Cray-HPE/docs-csm/pull/3633

**Pull request merged**